index.html

```diff
@@ -158,18 +190,74 @@ <h4>LET’S CONNECT</h4>
               <a href="N/A"> <img src="images/Vector (3).png" /> </a>
             </li>
           </ul>
-          <span><h4>Get my resume</h4></span>
+          <span>Get my resume</span>
+        </div>
+        <div id="Languages">
+          <span
+            ><h3>Languages</h3>
+            <img src="images/down-button.png"
+          /></span>
+          <ul id="languages-list">
+            <li>
+              <img src="images/js-icon.png" />
+              <h3>javaScript</h3>
+            </li>
+            <li>
+              <img src="images/html-icon.png" />
+              <h3>HTML</h3>
+            </li>
+            <li>
+              <img src="images/css-icon.png" />
+              <h3>CSS</h3>
+            </li>
+          </ul>
+          <span
+            ><h3>Frameworks</h3>
+            <img src="images/right-button.png"
+          /></span>
+          <hr />
+          <span
+            ><h3>Skills</h3>
+            <img src="images/right-button.png"
+          /></span>
+          <hr />
+        </div>
       </section>
-      <section id="Languages">
-            <span><h3>Languages</h3><img src="images/down-button.png"></span>
-               <ul id="languages-list" >
-                  <li><img src="images/js-icon.png"><H3>javaScript</H3></li>
-                  <li><img src="images/html-icon.png"><h3>HTML</h3></li>
-                   <li><img src="images/css-icon.png"><h3>CSS</h3></li>
-               </ul>
-               <span><h3>Frameworks</h3><img src="images/right-button.png"></span> <hr>
-               <span><h3>Skills</h3><img src="images/right-button.png"></span> <hr>
-    </section>
+
+      <footer class="contact-form">
+        <div class="text-container">
+          <h2 class="primary-text">Contact me</h2>
+          <p class="paragraph-details">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form action="https://formspree.io/f/xyyvbnal" method="post">
+          <input
+            type="text"
+            placeholder="Enter name..."
+            required
+            maxlength="30"
+            name="user_name"
+          />
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="user_email"
+            required
+          />
+          <textarea
+            cols="30"
+            rows="10"
+            maxlength="500"
+            name="user_message"
+            placeholder="Write your message here"
+            required
+          ></textarea>
+          <button type="submit" class="form-btn">Get in Touch</button>
+        </form>
+      </footer>
```

Comment:
- [ ] Great job working on the footer section but I think you forget to add the background image ✅ so kindly take it from the Figma design and put it as your background image ✅📌🚀🔥


-  Your design
![image](https://user-images.githubusercontent.com/58553711/200137767-572d091b-5b36-411f-87ae-f0a3c1f8e07b.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/200137782-d848c226-b13f-4133-9841-e1e16ed1a4a6.png)